src/System.Reflection.DispatchProxy/src/System/Reflection/DispatchProxyGenerator.cs

```diff
@@ -60,7 +60,7 @@ internal static class DispatchProxyGenerator
         // Proxy instances are not cached.  Their lifetime is entirely owned by the caller of DispatchProxy.Create.
         private static readonly Dictionary<Type, Dictionary<Type, Type>> s_baseTypeAndInterfaceToGeneratedProxyType = new Dictionary<Type, Dictionary<Type, Type>>();
         private static readonly ProxyAssembly s_proxyAssembly = new ProxyAssembly();
-        private static readonly MethodInfo s_dispatchProxyInvokeMethod = typeof(DispatchProxy).GetTypeInfo().GetDeclaredMethod("Invoke");
+        private static readonly MethodInfo? s_dispatchProxyInvokeMethod = typeof(DispatchProxy).GetTypeInfo().GetDeclaredMethod("Invoke");
```

Comment:
presumably we know this line will always succeed, since we're defining the Invoke method ourselves, so the nullness can be banged away, instead of asserted lower down.

Reply:
seems you're doing that with `s_delegateInvoke `

Reply:
> presumably we know this line will always succeed, since we're defining the Invoke method ourselves, so the nullness can be banged away, instead of asserted lower down.

True, but the assertion was already there which was the only usage. So nulling was safe and as it is not exposed private field null/non null doesn't really matter. Anyways from correctness side yes, banging is better